packages/core/saved-objects/core-saved-objects-migration-server-internal/src/document_migrator/document_migrator.ts

```diff
@@ -68,42 +29,49 @@ interface DocumentMigratorOptions {
 }
 
 /**
- * Manages migration of individual documents.
+ * Manages transformations of individual documents.
  */
 export interface VersionedTransformer {
-  migrationVersion: SavedObjectsMigrationVersion;
-  migrate: MigrateFn;
-  migrateAndConvert: MigrateAndConvertFn;
```

Comment:
`VersionedTransformer` was 'leaking' `migrationVersion` and `prepareMigrations` that shouldn't be present on that kind of publish-ish interfaces, so I cleaned that up